### PR TITLE
fix runner labels generation 

### PR DESCRIPTION
**Pull request by skarzi**
# Pull Request Template

## Description

This PR fixes issue with wrong label - `[]` - added for a self-hosted GitHub Actions Runner when using `github_actions_runner` ansible role with default `runner_labels` value (`[]`).

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

I have tried to test it locally with `molecule`, however, it's quite complicated, so please consider it untested locally, and let's rely on CI

### CODE REVIEW

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@
     - name: Check Runner
       assert:
         that:
+          - registered_runners.json.runners|json_query('labels[*].name')|flatten|sort == ['Linux', 'self-hosted', 'x86']
```
Comment (skarzi) [inline molecule/default/verify.yml:33]: I wanted to test if only default labels are used by default (no value for `runner_labels` provided), but I don't know how run this test :(

**Comment by skarzi** [comment]
GitHub UI with useless `[]` label added

2021-03-25-181057_196x27_scrot

**Comment by MonolithProjects** [comment]
Thx for your PR! 🙂 I will look on it bit later.

**Comment by skarzi** [comment]
And thank you for developing a really useful ansible role!

**Comment by MonolithProjects** [comment]
@skarzi your fix can be found in release [1.8.1](https://github.com/MonolithProjects/ansible-github_actions_runner/releases/tag/1.8.1) and it is available in Ansible Galaxy. Thanks for your help.

**Pull request merged**